src/components/invoices/preview/download-pdf-button.tsx

```diff
@@ -1,11 +1,19 @@
 "use client";
-import React, { useEffect, useState } from "react";
-
-import { PDFDownloadLink } from "@react-pdf/renderer";
+import React from "react";
 
 import { type InsertOrganizationType } from "@/lib/db/schema/organization";
 import { type InvoiceFormType } from "@/lib/form-schema/invoice-form";
 
+// Import via relative node_modules path to avoid "Element type
+// is invalid" error in the terminal when importing directly from
+// @react-pdf/renderer in a Next.js app
+//
+// ```
+// Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined
+// ```
+//
+// - https://github.com/diegomura/react-pdf/issues/2890#issuecomment-2443831013
+import { PDFDownloadLink } from "../../../../node_modules/@react-pdf/renderer/lib/react-pdf.browser";
```

Comment:
this change is unfortunate, but I hope that it's only temporary and that I'll be able to switch back to the `@react-pdf/renderer` import path soon, eg. possibly after this:

- https://github.com/karlhorky/invoiceo/issues/5